src/style-stars--3.css

```diff
@@ -0,0 +1,3 @@
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(../stars/yellow_star.png);
+}
```

Comment:
This styling for `.stars--3 .stars__star:nth-child(-n + 3)` is correct according to the checklist item that discourages the use of tag names for styling, with exceptions for `html` and `body`. You've correctly used classes to apply styles, which is good for maintainability and specificity.